src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Select, SelectOption,
+} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+
+/**
+ *
+ * @returns {object} Returns all of available cameras
+ */
+
+const getAvailableCameras = async (setDevices) => {
+  const cameraDevices = await navigator.mediaDevices.enumerateDevices()
+    .then((cameras) => cameras.filter((camera) => camera.kind === 'videoinput'));
+
+  setDevices([...cameraDevices]);
```

Comment:
this function does not seem to return the camera, it just calls the provided callback (`setDevices`) with the camera list as a parameter.

pls change this function to return the camera list, and use the `setDevices` function in the component itself.